// a.FullName с полями имя, фамилия, отчество(отчество опционально).
// b.Student с полями полное имя, массив оценок, средний балл.
// c.StudentsGroup с полем массив студентов.

syntax = "proto3";
package HW_7;

message Student {
	message FullName {
		optional string firstName = 1;
		optional string lastName = 2;
		optional string midleName = 3;
	}
	repeated FullName nameStudent = 1;
	repeated int32 marksStudent = 2;
	optional float averageMark = 3;
}

message StudentsGroup{
	repeated Student students = 1;
}